de4dot.code/deobfuscators/Dotfuscator/CflowDecrypter.cs

```diff
@@ -0,0 +1,74 @@
+﻿/*
+    Copyright (C) 2011-2015 de4dot@gmail.com
+
+    This file is part of de4dot.
+
+    de4dot is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    de4dot is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with de4dot.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+using System.Collections.Generic;
+using dnlib.DotNet;
+using dnlib.DotNet.Emit;
+using de4dot.blocks;
+
+namespace de4dot.code.deobfuscators.Dotfuscator {
+   
+
+    class CflowDecrypter {
+		ModuleDefMD module;
+		
+
+		public CflowDecrypter(ModuleDefMD module) {
+			this.module = module;
+		}
+        public void CflowClean()
+        {
+            foreach (TypeDef type in this.module.GetTypes())
```

Comment:
This whole file is still not formatted. It should use tabs and not spaces. Less important: there are a lot of empty lines for no reason.

Reply:
ok, I will check it.